src/formpack/utils/flatten_content.py

```diff
@@ -224,3 +224,13 @@ def _flatten_survey_row(row):
                 row['type'] = '{} {} or_other'.format(_type, _list_name)
             else:
                 row['type'] = '{} {}'.format(_type, _list_name)
+
+    # TODO: remove this once https://github.com/XLSForm/pyxform/issues/236 is
+    # fixed?
+    try:
+        _order = row['order']
+    except KeyError:
+        pass
+    else:
+        if not isinstance(_order, basestring):
```

Comment:
@jnm 
basestring is not compatible with Python3. I would use 

```
import six
...
    else:
          if not isintance(s, six.string_types):
```

Otherwise, if six is not available (or we don't want to add another dependency).
```
import sys

if sys.version_info[0] >= 3:
    basestring = str

...

   else:
        if not isinstance(_order, basestring):
```